user_guide.md

```diff
@@ -77,6 +83,36 @@ At any time you can inspect the kubernetes objects definitions for a particular
 ks show ${KF_ENV} -c kubeflow-core
 ```
 
+### Usage Reporting
+
+By default, Kubeflow will report **anonymous** usage data using [spartakus](https://github.com/kubernetes-incubator/spartakus), Kubernetes' reporting tool. Spartakus **does not report any personal information**. See [here](https://github.com/kubernetes-incubator/spartakus) for more detail.
+This is entirely voluntary and you can opt out by doing the following
```

Comment:
considering the opt-in/out discussion, i think we should just say "and you can disable reporting by doing the following"